### PR TITLE
Project: Testing Practice: Fix indentations of last item

### PR DESCRIPTION
**Pull request by AntoineLavacquery**
## Because
The indentation of the last function and the following JS code block seems to be wrong.

## This PR
- [x] Indentation removed from the last item (for analyzeArray function).

## Issue
Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
Thanks, good spot - looks like there was an accidental indent shift from the last PR that was missed.

In addition to the couple of extras below, would you mind also changing all of the list items' numbers to `1.` please? Just while there are extra changes to be made, we can knock this one out too. As per our [style guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#lazy-numbering), we use lazy numbering for ordered lists and this is one of the older lessons that haven't been updated to the current numbering style we use.

If you could do this as well, that'd be absolutely amazing :pray: Let me know if you run into any issues.

**Comment by AntoineLavacquery** [comment]
@MaoShizhong Thanks for your answer ! It's my first contribution, and I was actually following the page as a student (good news, Odin Project works :).
I have a basic github question : can I add your suggestion to batch for the next commit AND add the lazy numbering at the same time ? Or do I need to make 1 commit with your suggestion + 1 commit for the numbering ?

**Comment by MaoShizhong** [comment]
@AntoineLavacquery if you were to use the "commit suggestion" button, it will only apply the suggestion displayed in an individual commit.
If you make the change and commit manually, you can do it how you wish so just commit how you feel is most intuitive.

File: javascript/testing_javascript/project_testing_practice.md [hunk 1]
```diff
@@ -20,18 +20,18 @@ Write tests for the following, and then make the tests pass!
...
+5. An `analyzeArray` function that takes an array of numbers and returns an object with the following properties: `average`, `min`, `max`, and `length`.
+
+   ```javascript
+   const object = analyzeArray([1,8,3,4,2,6]);
+
+   object == {
+   average: 4,
+   min: 1,
+   max: 8,
+   length: 6
```
[the remark this answers is not in the document]
Comment (AntoineLavacquery) [reply javascript/testing_javascript/project_testing_practice.md]: Changes applied in commit fe5e209.

File: javascript/testing_javascript/project_testing_practice.md [hunk 2]
```diff
@@ -20,18 +20,18 @@ Write tests for the following, and then make the tests pass!
...
+   };
+
```
[the remark this answers is not in the document]
Comment (AntoineLavacquery) [reply javascript/testing_javascript/project_testing_practice.md]: Changes applied in commit fe5e209.

**Review by MaoShizhong — APPROVED** [review]
Thank you very much! :rocket:

**Pull request merged**